source/FreeRTOS_ARP.c

```diff
@@ -1338,6 +1340,24 @@ static BaseType_t prvFindCacheEntry( const MACAddress_t * pxMACAddress,
             }
         }
     }
+
+/*-----------------------------------------------------------*/
+
+/**
+ * @brief Create and send an ARP request packet.
+ *
+ * @param[in] ulIPAddress A 32-bit representation of the IP-address whose
+ *                         physical (MAC) address is required.
+ */
+    void FreeRTOS_OutputARPRequest( uint32_t ulIPAddress )
```

Comment:
It should enclosed with ipconfigIPv4_BACKWARD_COMPATIBLE flag.

Reply:
Hey Monika, `FreeRTOS_OutputARPRequest` is used in the stack when there isn't an endpoint readily available and have to use `FreeRTOS_FindEndPointOnNetMask` (basically what `FreeRTOS_OutputARPRequest` does). Hence, it is not a function kept for backward compatibility.

Reply:
Make sense.